include/vara/Feature/FeatureModel.h

```diff
@@ -21,7 +23,7 @@ class FeatureModel {
 public:
   using FeatureMapTy = llvm::StringMap<std::unique_ptr<Feature>>;
   using OrderedFeatureTy = OrderedFeatureVector;
-  using ConstraintTy = llvm::SmallVector<std::pair<std::string, bool>, 3>;
+  using ConstraintTy = Constraint;
   using ConstraintsTy = std::vector<ConstraintTy>;
```

Comment:
```suggestion
  using ConstraintContainerTy = std::vector<ConstraintTy>;
```
I think this could be a better name for what we do with the type.